### PR TITLE
Clean up and initial work on adding datetime field to the db.

### LINKED ISSUE #1: Exercise 3: Records
In this exercise, you'll work with the beginnings of a database-like structure. The program you're working toward is a customer contact list that tracks basic information about customers and your connections with them.

To start, make a program that:
-   Prompts the user to enter the following information for a customer contact list:
    -   Last name
    -   First name
    -   Phone number
    -   Email address
-   Stores the supplied information in a dictionary with appropriately named keys
-   Prints out the information in a format that looks good to you

### PR DESCRIPTION
**Pull request by datapawn**
Started work with DateTime and added initial thoughts on UI for adding Contact with a Client
Also added a faux record to play with, so you don't have to make one first each time.

### CODE REVIEW

**Comment by datapawn** [comment]
Changed Datetime to Date.  I'm still working on this, don't review yet.

File: Exercises/ex-03/r1/main.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    #initialize starting input:  p
+    startnum = p
+    #initialize an empty string to catch the number we need.
+    outnum = ""
+    #goodnums is a list of just the digits we want to accept.
+    goodnums = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
+
+    #for loop goes through their string and adds only the digits of goodnums to the output variable
+    for num in startnum:
+        if num in goodnums:
```
Comment (JayLoomis) [inline Exercises/ex-03/r1/main.py]:
```suggestion
        if num.isdigit():
```

File: Exercises/ex-03/r1/main.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    print("SEARCH is not available at this time.\n\n")
+    main()
+
+def verifyphone(p):
+    #initialize starting input:  p
+    startnum = p
+    #initialize an empty string to catch the number we need.
+    outnum = ""
+    #goodnums is a list of just the digits we want to accept.
+    goodnums = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
```
Comment (JayLoomis) [inline Exercises/ex-03/r1/main.py]:
```suggestion
```

**Review by JayLoomis — CHANGES_REQUESTED** [review]
A simple change to test the process of accepting changes.

**Comment by JayLoomis** [comment]
LGTM

**Pull request merged**